content/docs/operating/security.md

```diff
@@ -161,3 +161,10 @@ members of the Prometheus development team and the staff of those providers
 have access. If you are concerned about the exact provenance of your binaries,
 it is recommended to build them yourself rather than relying on the
 pre-built binaries provided by the project.
+
+## External audits
+
+[CNCF](https://cncf.io) sponsored an external security audit by
```

Comment:
Should we summarise this here?

Reply:
I think yes. Attribution is important in both FLOSS & security and I would like to acknowledge the fact that CNCF spent money on us.

Reply:
I meant more, that we should add a few sentences summarising the report.

Reply:
Done.